adapters/repos/backups/backup_manager_integration_test.go

```diff
@@ -77,18 +87,31 @@ func TestBackupManager_CreateBackup(t *testing.T) {
 		ctx, cancel := testCtx()
 		defer cancel()
 
+		painters, paintings := createPainterClass(), createPaintingsClass()
+		harness := setupTestingHarness(t, ctx, painters, paintings)
+
 		compose := docker.New()
 		compose.WithGCS()
 		container, err := compose.Start(ctx)
 		require.Nil(t, err)
-		require.Nil(t, os.Setenv("GCS_ENDPOINT", container.GetGCS().URI()))
-		require.Nil(t, os.Setenv("STORAGE_EMULATOR_HOST", container.GetGCS().URI()))
-		require.Nil(t, os.Setenv("GOOGLE_APPLICATION_CREDENTIALS", ""))
-		require.Nil(t, os.Setenv("GOOGLE_CLOUD_PROJECT", "project-id"))
 
-		painters, paintings := createPainterClass(), createPaintingsClass()
+		endpoint := container.GetGCS().URI()
+		bucketName := "weaviate-snapshots"
+		projectID := "project-id"
 
-		harness := setupTestingHarness(t, ctx, painters, paintings)
+		t.Run("setup env", func(t *testing.T) {
```

Comment:
I would recommend to move all `gcs` and `s3` tests to `test/modules` and only leave here the tests for `filesystem` module. I think that's a better place where we specifically deal with `modules` and test them